sap-s4hana-cloud/2.2/antora.yml

```diff
@@ -0,0 +1,16 @@
+name: sap-s4hana-cloud-connector
+version: '2.2'
+display_version: 2.2 (Mule 4)
+title: SAP S/4HANA OData Cloud Connector
```

Comment:
Don't we need to remove Cloud word also from this title?

Reply:
Yes, thanks!